### PR TITLE
add checkstyle

### PR DESCRIPTION
**Pull request by arufanov**
Checkstyle plugin has been added

### CODE REVIEW

**Comment by arufanov** [comment]
@yurichervoniy 
* lets discuss.

File: pom.xml [hunk 1]
```diff
@@ -80,6 +83,12 @@
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (yurichervoniy) [inline pom.xml:86]: Why in dependicies?

File: pom.xml [hunk 2]
```diff
@@ -153,5 +190,31 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>${checkstyle.plugin.version}</version>
+                <configuration>
+                    <configLocation>checkstyle.xml</configLocation>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jxr-plugin</artifactId>
```
Comment (yurichervoniy) [inline pom.xml:209]: What is this?

File: pom.xml (hunk 1 above)
Comment (yurichervoniy) [reply pom.xml]: Remove

**Comment by arufanov** [comment]
task:
* remove package info 
* reformat all code after merge.

**Review by yurichervoniy — APPROVED** [review]

**Pull request merged**